src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksPanel.qml

```diff
@@ -41,73 +46,99 @@ Item {
         property string currentItemNavigationName: ""
     }
 
-    ColumnLayout {
-        id: contentColumn
+    RowLayout {
 
         anchors.fill: parent
 
-        readonly property int sideMargin: 12
-        spacing: sideMargin
+        TrackEffectsPanel {
+            id: trackEffectsPanel
+            Layout.preferredWidth: root.effectsPanelWidth
+            Layout.maximumWidth: root.effectsPanelWidth
+            Layout.minimumWidth: root.effectsPanelWidth
+            Layout.fillHeight: true
+            visible: false
+        }
+
+        SeparatorLine {
+            id: separatorLine
+            orientation: Qt.Vertical
```

Comment:
these properties are already set by default for the component, so they are not needed here